original/java/src/test/java/PageDBTest.java

```diff
@@ -0,0 +1,12 @@
+import org.dbpedia.categories.database.PageDB;
+
+public class PageDBTest {
+
+	public static void main(String[] argv){
```

Comment:
I am reading about Unit Testing and will complete this part by next commit in a day or two.